app/indexers/concerns/iiif_print/child_indexer.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module IiifPrint
+  module ChildIndexer
+    extend ActiveSupport::Concern
+    include IiifPrint::IiifPrintBehavior
+
+    def generate_solr_document
+      super.tap do |solr_doc|
+        solr_doc['is_child_bsi'] = object.is_child
+        solr_doc['is_page_of_ssim'] = ancestor_ids(object)
+        solr_doc['file_set_ids_ssim'] = all_decendent_file_sets(object)
+      end
+    end
+
+    private
+
+    ##
+    # relationship indexing for fileset and works
+    #
+    # @param options [Hash] options hash provided by Blacklight
+    # @return [String] snippets HTML to be rendered
+    # rubocop:disable Rails/OutputSafety
+    def ancestor_ids(o)
```

Comment:
if this method is already in `IiifPrint::IiifPrintBehavior` does it need to be here again?

Reply:
At the time, I thought that it wasn't hitting this method when I put it in the mixin. Maybe something was off with my env then. Thanks for testing it, I'll try it again. Ideally, since this is a shared method it should go in the behavior mixin.